projects/redis/redis/Makefile

```diff
@@ -0,0 +1,25 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+
+REPO=redis
+REPO_OWNER=redis
+IMAGE_COMPONENT=redis/redis
+IMAGE_BUILD_ARGS?=IMAGE_TAG
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base-glibc
+SIMPLE_CREATE_BINARIES=false
+GO_MOD_DOWNLOAD_TARGETS=$(GIT_PATCH_TARGET)
+HAS_GO_LICENSES=false
+
+BUILD_TARGETS=local-images
```

Comment:
Can we remove these? Is a target running that wont work without these overrides?

Reply:
Done.